nextflow.config

```diff
@@ -59,7 +59,7 @@ params {
     affy_build_annotation    = true
 
     // Proteus-specific options
-    proteus_measurecol_prefix = 'LFQ intensity '
+    proteus_measurecol_prefix = 'LFQ intensity'
```

Comment:
```suggestion
    proteus_measurecol_prefix = 'LFQ intensity '
```

Reply:
That space was not accidental, it's necessary :)

Reply:
Ahh, OK, blame @sateeshperi ;-)